.buildkite/hooks/pre-command

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+set -euo pipefail
+
+DOCKER_REGISTRY_SECRET_PATH="kv/ci-shared/platform-ingest/docker_registry_prod"
+
+if [[ "$BUILDKITE_PIPELINE_SLUG" == "temp-fleet-server" && "$BUILDKITE_STEP_KEY" == "publish" ]]; then #should be changed to "fleet-server" after tests
```

Comment:
Where is the `temp-fleet-server` defined?

Reply:
It's an environment variable. 
I removed this extra option ($BUILDKITE_PIPELINE_SLUG) because there is we have only one fleet-server pipeline and we don't need to verify it. We need to define only step.